docs/charon/charon-cli-reference.md

```diff
@@ -107,7 +107,7 @@ Flags:
       --fee-recipient-address string   Optional Ethereum address of the fee recipient
   -h, --help                           Help for cluster
       --insecure-keys                  Generates insecure keystore files. This should never be used. It is not supported on mainnet.
-      --keymanager-addresses strings   Comma separated list of keymanager URLs to push validator key shares to. Note that multiple addresses are required, one for each node in the cluster, with node0's keyshares being pushed to the first address, node1's keyshares to the second, and so on.
+      --keymanager-addresses strings   Comma separated list of keymanager URLs to import validator key shares to. Note that multiple addresses are required, one for each node in the cluster, with node0's keyshares being imported to the first address, node1's keyshares to the second, and so on.
```

Comment:
we are `exporting` key shares TO these addresses and not `importing` FROM these addresses
```suggestion
      --keymanager-addresses strings   Comma separated list of keymanager URLs to export validator key shares to. Note that multiple addresses are required, one for each node in the cluster, with node0's keyshares being exported to the first address, node1's keyshares to the second, and so on.
```